eng/pipelines/templates/steps/dependency.tests.yml

```diff
@@ -23,12 +23,21 @@ parameters:
 - name: ServiceDirectory
```

Comment:
We might want to rename this template to something like split.dependencies.yml

Reply:
Nevermind given we are plugging this in as a set of tests I think we can keep the name dependency.tests.yml.